Demo.xcodeproj/project.pbxproj

```diff
@@ -23,25 +22,11 @@
 		22BFC030237D7AEC00B55C5E /* IndicatorLabel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 22BFC02F237D7AEC00B55C5E /* IndicatorLabel.swift */; };
 		22BFC032237D852900B55C5E /* RoomTableViewCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 22BFC031237D852900B55C5E /* RoomTableViewCell.swift */; };
 		22BFE7FD23DF24090073E6D4 /* Environment.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 22BFE7FC23DF24090073E6D4 /* Environment.framework */; };
-		22BFE7FE23DF24090073E6D4 /* Environment.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 22BFE7FC23DF24090073E6D4 /* Environment.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		22BFE80023DF25640073E6D4 /* Environment+Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 22BFE7FF23DF25640073E6D4 /* Environment+Extensions.swift */; };
 		22FDE9C8237B2127005F13CA /* UIBezierPath+Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 22FDE9C7237B2127005F13CA /* UIBezierPath+Extensions.swift */; };
+		735A834A23EB3B2E0094086A /* PusherChatkit in Frameworks */ = {isa = PBXBuildFile; productRef = 735A834923EB3B2E0094086A /* PusherChatkit */; };
 /* End PBXBuildFile section */
 
-/* Begin PBXCopyFilesBuildPhase section */
-		223D2C1E238EA5ED007B0BC6 /* Embed Frameworks */ = {
```

Comment:
Why `Environment` framework is no longer embedded?

Reply:
This change was made by Xcode automatically.  I was hoping to test whether an embedded framework was required or not but I have been unable to unlock my iPhone so I'm unable to test 🤦🏼‍♂️. In the interests of merging this PR I have reverted this change.

Reply:
It looks like this is some old reference in the project file and that is why Xcode automatically removed it. However, `Environment` framework is not embedded in the demo app and the app is crashing on the device (I just tested this on my phone). Let's fix that in a separate PR.

Reply:
👍  thanks for the info. Thought it might crash :-(